src/js/classes/AjaxBootstrapSelectList.js

```diff
@@ -284,6 +284,12 @@ AjaxBootstrapSelectList.prototype.replaceOptions = function (data) {
 
     // Replace the options.
     this.plugin.$element.html(output);
+
+    // Collapse dropdown after large search results
+    this.plugin.$element.nextAll('.dropdown-menu')
+        .find('ul.dropdown-menu.inner')
```

Comment:
This traverses the DOM each time. DOM elements should get their own property on the object that can be targeted for performance reasons.